tvb_scripts/io/h5_reader_base.py

```diff
@@ -0,0 +1,47 @@
+# -*- coding: utf-8 -*-
```

Comment:
Are these really used?
It will help us evaluate better the current state if we find for usage also.